### DISCUSSION TITLE
MarqueeTextQueue control

### DISCUSSION THREAD

**Discussion by Avid29** [CommunityToolkit/Labs-Windows#550] [Ideas]
Currently the `MarqueeText` control only supports looping handling a single string. This is especially inconvenient with the new Cycling mode, which was initially planned to cycle between multiple strings but instead simply plays an animation when swapping text.

That behavior was implemented with this control in mind. This control would wrap the `MarqueeText` control and would track the Storyboard events to swap the text at the end of animations. The goals of this control would be the following:

- Autonomously cycle between multiple strings in `Cycle` mode
- Seamlessly enqueue text in `Ticker` or `Loop` mode

**Comment by Arlodotexe** [comment]
As discussed in call, we should consider https://github.com/CommunityToolkit/Labs-Windows/issues/551#issue-2340986935 since it handles this scenario out of the box.
